itl/command-plugins.conf

```diff
@@ -549,6 +549,175 @@ object CheckCommand "http" {
+		}
+		"--ca-cert" = {
+			value = "$curl_ca-cert-file$"
+			description = "CA certificate file to verify peer against"
+		}
+		"-D" = {
+			set_if = "$curl_verify-peer-cert$"
+			description = "Verify the peer's SSL certificate and hostname"
+		}
+		"-e" = {
+			value = "$curl_expext-string$"
+			description = "Comma-delimited list of strings, at least one of them is expected in the first (status) line of the server response (default: HTTP/), If specified skips all other status line logic (ex: 3xx, 4xx, 5xx processing)"
+		}
+		"-d" = {
+			value = "$curl_expect-header-string$"
+			description = "String to expect in the response headers"
+		}
+		"-s" = {
+			value = "$curl_expect-content-string$"
+			description = "String to expect in the content"
+		}
+		"-u" = {
+			value = "$curl_url$"
+			description = "URL to GET or POST (default: /)"
+		}
+		"-P" = {
+			value = "$curl_post-data$"
+			description = "URL encoded http POST data"
+		}
+		"-j" = {
+			value = "$curl_http_method$"
+			description = "Set HTTP method (for example: HEAD, OPTIONS, TRACE, PUT, DELETE, CONNECT)"
+		}
+		"-N" = {
+			value = "$curl_no-body$"
+			description = "Don't wait for document body: stop reading after headers. (Note that this still does an HTTP GET or POST, not a HEAD.)"
+		}
+		"-M" = {
+			value = "$curl_max-age$"
+			description = "Warn if document is more than SECONDS old. the number can also be of the form '10m' for minutes, '10h' for hours, or '10d' for days."
+		}
+		"-T" = {
+			value = "$curl_content-type$"
+			description = "specify Content-Type header media type when POSTing"
+		}
+		"-l" = {
+			value = "$curl_linespan$"
+			description = "Allow regex to span newlines (must precede -r or -R)"
+		}
+		"-r" = {
+			value = "$curl_ereg$"
+			description = "Search page for regex STRING"
+		}
+		"-R" = {
+			value = "$curl_eregi$"
+			description = "Search page for case-insensitive regex STRING"
+		}
+		"--invert-regex" = {
+			set_if = "$curl_invert_regex$"
+			description = "When using regex, return CRITICAL if found, OK if not"
+		}
+		"-a" = {
+			value = "$curl_authorization$"
+			description = "Username:password on sites with basic authentication"
+		}
+		"-b" = {
+			value = "$curl_proxy-authorization$"
+			description = "Username:password on proxy-servers with basic authentication"
+		}
+		"-A" = {
+			value = "$curl_user-agent$"
+			description = "String to be sent in http header as 'User Agent'"
+		}
+		"-k" = {
+			value = "$curl_header$"
+			repeat_key = true
+			description = "Any other tags to be sent in http header. Use multiple times for additional headers"
+		}
+		"-E" = {
+			set_if = "$curl_extended-perfdata$"
+			description = "Print additional performance data"
+		}
+		"-B" = {
+			set_if = "$curl_show_body$"
+			description = "Print body content below status line"
+		}
+		"-f" = {
+			value = "$curl_onredirect$"
+			description = "Options: <ok|warning|critical|follow|sticky|stickyport|curl> How to handle redirected pages."
```

Comment:
This description differs from the current man page.

Reply:
I have just seen 7a00d2e8ebfd2902a017134a45eb7d59107a64c7 and the discussion above. Sorry.